config/settings.yml

```diff
@@ -10,6 +10,20 @@ dttp:
 features:
   home_text: false
 
+personas:
```

Comment:
This one approach I took. Other suggestions are welcome.

Reply:
i think we opted into keeping this sort of seed data in `config/initializers`.